### PR TITLE
Added library support and updated documentation

### PR DESCRIPTION
**Pull request by AvidDabbler**
I found your repo and I wanted to use it as a library. I ended up making the functions public and exporting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+
+While the repo is named `gtfs-to-geojson` the crate is called `gtfs_geojson` and can me installed using 
+
+```bash
+cargo add gtfs_geojson
+```
+
+## Functions
+
+The `gtfs_geojson` library use the `gtfs_structures` library and there types to convert gtfs files over to geojson.
```
Comment (Tristramg) [inline README.md]:
```suggestion
The `gtfs_geojson` library use the `gtfs_structures` library and its types to convert gtfs files over to geojson.
```

**Review by Tristramg — CHANGES_REQUESTED** [review]
Thank you very much for this contribution!

Would you mind squashing your commits and bumping the version in `Cargo.toml` to 0.4.0?

Out of curiosity (no need to answer), what do you use this crate for?

File: README.md (hunk 1 above)
Comment (AvidDabbler) [reply README.md]: Sure thing! I love transit and work with GTFS a lot and I had built [gtfs-parser](https://github.com/AvidDabbler/gtfs-parser) recently, which is a project that is similar, but in typescript inspired by the out of date [gtfs2geojson](https://github.com/node-geojson/gtfs2geojson). I was looking for a drop in solution for a project called [simple-transit-map](https://github.com/ioTransit/simple-transit-map). 

Eventually I would like to use rust on client side alongside mapbox and maybe extend one what i have built with simple-transit-map to ingest gtfs directly on client.

**Comment by AvidDabbler** [comment]
Sure thing! I love transit and work with GTFS a lot and I had built [gtfs-parser](https://github.com/AvidDabbler/gtfs-parser) recently, which is a project that is similar, but in typescript inspired by the out of date [gtfs2geojson](https://github.com/node-geojson/gtfs2geojson). I was looking for a drop in solution for a project called [simple-transit-map](https://github.com/ioTransit/simple-transit-map).

Eventually I would like to use rust on client side alongside mapbox and maybe extend one what i have built with simple-transit-map to ingest gtfs directly on client.

I was not able to squash the changes. Something has happened on my computer where it is just complaining about how I cannot squash merges. This seems to be an open issue.

Issue
https://github.com/desktop/desktop/issues/13051

image

**Comment by Tristramg** [comment]
It’s ok, I forgot that github has a feature to squash itself.

Thank you for the contribution!

**Pull request merged**